app/controllers/api/v1/pets_controller.rb

```diff
@@ -0,0 +1,84 @@
+class Api::V1::PetsController < ApplicationController
+  def index
+    user = user_exist?(params[:user_id])
```

Comment:
Just a note for our future reference, this is where Auth will take place. A conditional on an active session or whatnot.